libs/coin-algorand/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@ledgerhq/coin-algorand",
+  "version": "0.1.0",
+  "description": "Ledger Algorand Coin integration",
+  "keywords": [
+    "Ledger",
+    "LedgerWallet",
+    "algo",
+    "Algorand",
+    "Hardware Wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LedgerHQ/ledger-live.git"
+  },
+  "bugs": {
+    "url": "https://github.com/LedgerHQ/ledger-live/issues"
+  },
+  "homepage": "https://github.com/LedgerHQ/ledger-live/tree/develop/libs/coin-algorand",
+  "publishConfig": {
+    "access": "public"
+  },
+  "typesVersions": {
+    "*": {
+      "lib/*": [
+        "lib/*"
+      ],
+      "lib-es/*": [
+        "lib-es/*"
+      ],
+      "*": [
+        "lib/*"
+      ]
+    }
+  },
+  "exports": {
+    "./lib/*": "./lib/*.js",
+    "./lib-es/*": "./lib-es/*.js",
+    "./*": {
+      "require": "./lib/*.js",
+      "default": "./lib-es/*.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@ledgerhq/coin-framework": "workspace:^",
+    "@ledgerhq/cryptoassets": "workspace:^",
+    "@ledgerhq/devices": "workspace:^",
+    "@ledgerhq/errors": "workspace:^",
+    "@ledgerhq/hw-app-algorand": "workspace:^",
+    "@ledgerhq/live-app-sdk": "^0.8.1",
```

Comment:
I didn't see that you need this dependency.

Reply:
Indeed, I might need to clean up the package.json to make sure there is no extra dep 👍